packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -112,3 +112,24 @@ Template locking allows locking the `InnerBlocks` area for the current template.
 If locking is not set in an `InnerBlocks` area: the locking of the parent `InnerBlocks` area is used.
 
 If the block is a top level block: the locking of the Custom Post Type is used.
+
+### `renderAppender`
+* **Type:** `String|Function`
+* **Default:** - `null`. By default the `auto-insert` behaviour (see below) is utilised.
+* **Options:** 
+    - `block` - uses the Block List Appender (`packages/block-editor/src/components/block-list-appender/index.js`) to display a "Add Block" message with an icon which fills the Block preview. No default Block is inserted.
+    - `auto-insert` - automatically inserts whichever block is configured as the default block via `wp.blocks.getDefaultBlockName` (typically `paragraph`).
+
+Determines the placeholder behaviour when the Block is initially inserted. You may pass a render function to provide your own placeholder as required.
+
+### `hideAppenderWhenChildren`
```

Comment:
Should we remove this prop now that we have InnerBlocks.HideWhenChildren.

Reply:
Docs updated